junebug/logging_service.py

```diff
@@ -111,6 +111,8 @@ def __init__(self, worker_id, path, rotate, max_files, logger=None):
         self.max_files = max_files
 
     def startService(self):
+        if not os.path.exists(self.path):
+            os.makedirs(self.path, 0777)
```

Comment:
Do we want 755 instead? Only read + exec for everyone, read, exec & write for the owner?